server/asset/btc/rpcclient.go

```diff
@@ -249,6 +284,22 @@ func (rc *RPCClient) GetBlockVerbose(blockHash *chainhash.Hash) (*GetBlockVerbos
 	return res, rc.call(methodGetBlock, anylist{blockHash.String(), arg}, res)
 }
 
+// GetBlockHash fetches the block hash for the block at the given index.
+func (rc *RPCClient) GetBlockHash(index int64) (*chainhash.Hash, error) {
+	var blockHashStr string
+	err := rc.Call(methodGetBlockHash, []interface{}{index}, &blockHashStr)
```

Comment:
Pattern is to use `rc.call`, the non-exported version.
Also `anylist{index}`, but it's the same.

Reply:
> Pattern is to use rc.call, the non-exported version.

Yeah, thanks for the review.